assets/tokens/rinkeby-token-list.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "IDO Rinkeby",
+  "version": {
+    "major": 0,
+    "minor": 0,
+    "patch": 1
+  },
+  "keywords": [
+    "IDO",
+    "Rinkeby",
+    "tokens",
+    "trusted"
+  ],
+  "logoURI": "https://avatars.githubusercontent.com/u/24954468?s=200&v=4",
+  "timestamp": "2021-03-15T11:27:16.709Z",
+  "tokens": [
+    {
+      "name": "Gnosis",
+      "chainId": 4,
+      "symbol": "GNO",
+      "decimals": 18,
+      "address": "0xd0Dab4E640D95E9E8A47545598c33e31bDb53C7c",
+      "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0x6810e776880C02933D47DB1b9fc05908e5386b96/logo.png"
```

Comment:
`logoURI` could be pre-calculated on the code.
Also we could use the Gnosis Safe logo endpoints `https://gnosis-safe-token-logos.s3.amazonaws.com/{}.png`